MathlibTest/CategoryTheory/ToFun.lean

```diff
@@ -0,0 +1,9 @@
+import Mathlib.Tactic.ToFun
```

Comment:
Why is this test inside `CategoryTheory`? A top-level file seems perfectly fine to me.